kitsune/sumo/static/sumo/js/ajaxvote.js

```diff
@@ -73,7 +73,7 @@ export default function AjaxVote(form, options) {
             },
             error: function() {
               var msg = gettext('There was an error submitting your vote.');
-              self.showMessage(msg, $btn);
+              self.showMessage(msg, $btn, $form);
```

Comment:
This fixes the JS error that caused the spinner to be displayed forever instead of displaying the error message when votiing, and was a symptom of the larger CSRF token error.